jQuery/index.html

```diff
@@ -9,12 +10,20 @@
     <div id="hh-menu">
```

Comment:
I'm not a fan of too many id's on a page, for what it's worth. I get that you'll have a single menu, and so an ID is fine, but you really don't have to qualify it with a namespace (`hh-`), we know that this page is the homepage for honey headquarters.